src/libs/udq_helper_utils/udq_utils/udq_models.py

```diff
@@ -13,7 +13,7 @@ class EntityComponentPropertyRef():
     consists of an entityId, componentName, and propertyName
     """
 
-    def __init__(self, entity_id: str, component_name: str, property_name: str):
+    def __init__(self, property_name: str, entity_id: str = None, component_name: str = None):
```

Comment:
Is there any reason we change the input parameters order? Means we also need to change all the calling places (I assume the only one is L178 one?)

Reply:
Yes the only place this is called is L178 and L180. `property_name` is always required while `entity_id` and `component_name` are optional. Optional parameters should not be before required ones.